.github/workflows/reusable_checks.yml

```diff
@@ -52,6 +52,14 @@ jobs:
     - name: Check Python formatting
       run: cmake --build build --target black-format-check
 
+    - name: Run check-license
+      run: |
+        bash ./check_license/check_headers.sh . "Apache-2.0 WITH LLVM-exception" -v
```

Comment:
so, now this script has proper execute permissions, we can omit `bash` at the beginning of the line...?